dashboard/app/controllers/tracking_pixel_controller.rb

```diff
@@ -0,0 +1,22 @@
+require 'cdo/firehose'
+
+class TrackingPixelController < ApplicationController
+  def index
+    prevent_caching
+    curriculum_page = URI.unescape(params[:from]).split('/').reverse.drop(1).reverse.join('/')
```

Comment:
It's removing the random number that we generated and appended on the curriculum builder side.

Reply:
Oh! I think we can avoid this if, instead of appending the random number to the `from` parameter, we make it a separate query parameter. i.e. Instead of `?from=<page><random>` we use `?from=<page>&r=<random>`.  If we do that, you shouldn't need to remove the random number manually here.

Reply:
doh. yep. That's way better. Updating...